src/lib/parse/CompoundValueNode.h

```diff
@@ -127,6 +126,7 @@ class CompoundValueNode
   void                check(void);
   std::string         finish(void);
   std::string         render(Format format, const std::string& indent);
+  std::string         renderV2(void);
```

Comment:
So, I change all 'renderV2' to 'toJson' and the 'render' method might disappear ...